Sandbox Embed Modal updates (#164)
apollographql/docs: src/content/studio/explorer/sandbox.mdx

     width="600"
   />

> 3. Still in the dialog, use the tabs above the code snippet to select the snippet for your use case:

Reviewer: ```suggestion
3. Within the dialog, use the tabs to select the code snippet for your use case:
```
